context/AuthContextProvider.test.tsx

```diff
@@ -55,11 +63,36 @@ describe('AuthContextProvider', () => {
 
     mockCreateEmailPasswordSession.mockRejectedValue(mockError);
 
-    const error = await loginAccount({user, router, getUser});
+    const error = await loginAccount({ user, router, getUser });
+
+    expect(error).toEqual(mockError);
+    expect(toast.custom).toHaveBeenCalledWith(
+      <Alert variant={AlertVariants.Error} message="Something went wrong!" />,
+    );
+  });
+
+  //logout tests
+  test('after a successful logout it shows success notification', async () => {
+    await logoutAccount({ resetUser, setIsSignedIn, router });
+
+    expect(setIsSignedIn).toHaveBeenCalledWith(false);
+    expect(resetUser).toHaveBeenCalled();
+    expect(router.push).toHaveBeenCalledWith('/login');
+    expect(toast.custom).toHaveBeenCalledWith(
+      <Alert variant={AlertVariants.Success} message="Logged Out" />,
+    );
+  });
+
+  test('after logout attempt errors it shows error notification', async () => {
```

Comment:
There seems to be some inconsistency with double and single quotes here. Please update this test so it uses one or the other.

Reply:
I thinks it's caused by my prettier settings. I'll see if I can get these fixed